cnf-certification-test/identifiers/remediation.go

```diff
@@ -195,4 +195,6 @@ const (
 	CPUIsolationRemediation = `CPU isolation testing is enabled.  Please ensure that all pods adhere to the CPU isolation requirements`
 
 	UID1337Remediation = `Use another process UID that is not 1337`
+
+	AffinityRequiredRemediation = `If a pod/statefulset/deployment is required to use affinity rules, please add 'AffinityRequired: true' as a label`
```

Comment:
Same: I'd emphasise that label value must be string, not boolean, so I'd say `AffinityRequired: 'true'`, maybe.

Reply:
Done.